static/oas/v1/sports/components/logos.yml

```diff
@@ -35,17 +36,18 @@ paths:
           required: true
           description: The ID of the logo to retrieve
           schema:
-            type: string
+            type: integer
       responses:
         200:
-          description: The response body should contain the logo binary
+          description: The response body should contain the png logo
           content:
             multipart/form-data:
               schema:
                 type: object
```

Comment:
responses:
        '200':
          description: OK
          content:
              image/png:
                schema: 
                  type: string
                  format: binary

Reply:
Done

Reply:
Thanks for the feedback, I was unsure about what should go here

Reply:
@jak103 My bad on that, can you re-review? I've fixed the issue.
![image](https://github.com/jak103/powerplay/assets/24322375/09f67384-b65b-4fd3-8b82-a84e6285840b)
![image](https://github.com/jak103/powerplay/assets/24322375/f1aa8121-9945-4900-9188-3f0c27739771)